sdcm/cluster.py

```diff
@@ -4054,6 +4054,14 @@ def download_scylla_monitoring(self, node):
         if node.is_ubuntu():
             node.remoter.run(f'sed -i "s/python3/python3.6/g" {self.monitor_install_path}/*.py')
 
+    @staticmethod
+    def start_node_exporter(node):
+        start_node_exporter_script = dedent(f'''
+            docker run --restart=always -d --net="host" --pid="host" -v "/:/host:ro,rslave" --cap-add=SYS_TIME \
```

Comment:
This way to run a container is not good for `docker` backend.

Please, look at `sdcm/utils/remotewebbrowser.py` for an example how to run a Docker container on a remote node.
Also, please, add  a base mixin class for such containers to `sdcm/utils/docker.py` with cached property `_ssh_docker_client` or something like that or I can do this for you.

Reply:
What's wrong with running it like this ? And why do we want to cache it ? It's one docker that we want to have running and never stop, we don't need to handle it after this step at all. not to stop it or restart.

Reply:
This is true for backends which run separate VM for monitoring, but not for docker. Right now we use localhost as monitoring node. So, we need to take care about every container we run.

Reply:
It's not about caching "container", it's about caching DockerClient object.

Reply:
If we run this locally, this isn't needed at all, since no point running node_collect since we don't have a node, things are running locally.

Reply:
So, it requires to handle such things anyway, isn't it?

Reply:
Yes I'll need to see how to skip it if docker backend is used

Reply:
disabling calling this function, for docker backend